Src/GoogleApis.Core/Apis/Util/PageStreamer.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Done. Not sure whether the year should be 2015 or 2016 though, given that the original code was checked into Google.Api.Common in 2015...